### PR TITLE
Added error check in addHostIntent

### PR DESCRIPTION
**Pull request by hakrishn**

### CODE REVIEW

**Comment by jhall11** [comment]
Did you want to abandon this? I can't remember what we decided

**Comment by hakrishn** [comment]
Added error check in addHostIntent to return handle.... is required.

**Comment by jhall11** [comment]
We cherry picked that one into master didn't we?

**Comment by hakrishn** [comment]
Yes. Should be already in master.

File: TestON/drivers/common/cli/onosclidriver.py [hunk 1]
```diff
@@ -901,9 +901,12 @@ def addHostIntent( self, hostIdOne, hostIdTwo ):
                            str( hostIdOne ) + " and " + str( hostIdTwo ) )
                 return main.TRUE
 
+<<<<<<< HEAD
```
Comment (jhall11) [inline TestON/drivers/common/cli/onosclidriver.py]: Can you fix this merge conflict?

File: TestON/tests/IntentPerfNext/IntentPerfNext.params [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <PARAMS>
-    <testcases>1,4,5,2,3,4,5,2,3,4,5,2,3,4</testcases>
+    <testcases>1,4,5,4,5,4,5,4</testcases>
```
Comment (jhall11) [inline TestON/tests/IntentPerfNext/IntentPerfNext.params]: was this change intentional?

File: TestON/tests/IntentPerfNext/IntentPerfNext.py [hunk 3]
```diff
@@ -37,7 +37,8 @@ def CASE1( self, main ):
         main.ONOSbench.onosUninstall( nodeIp=ONOS4Ip )
         main.ONOSbench.onosUninstall( nodeIp=ONOS5Ip )
         main.ONOSbench.onosUninstall( nodeIp=ONOS6Ip )
```
Comment (jhall11) [inline TestON/tests/IntentPerfNext/IntentPerfNext.py]: and these perf test changes?

File: TestON/tests/IntentPerfNext/IntentPerfNext.params (hunk 2 above)
Comment (hakrishn) [inline TestON/tests/IntentPerfNext/IntentPerfNext.params]: This seems like the same issue that Shreya encountered last week. I did not change anything in the Perf scripts. Something is wrong with Git.

File: TestON/tests/IntentPerfNext/IntentPerfNext.py (hunk 3 above)
Comment (hakrishn) [inline TestON/tests/IntentPerfNext/IntentPerfNext.py]: I did not change any driver files after pulling Kelvins mininet changes sometime back. So, this commit should have only
OnosCHO file changes.

Comment (jhall11) [inline TestON/tests/IntentPerfNext/IntentPerfNext.py]: The commits are from andrew, so either you were both on the same machine,
or somehow you pulled them into your branch.

On Wed, Feb 18, 2015 at 8:04 PM, hakrishn <notifications@github.com> wrote:

> In TestON/tests/IntentPerfNext/IntentPerfNext.py
> <https://github.com/opennetworkinglab/ONLabTest/pull/63#discussion_r24964796>
> :
>
> > @@ -37,7 +37,8 @@ def CASE1( self, main ):
> >          main.ONOSbench.onosUninstall( nodeIp=ONOS4Ip )
> >          main.ONOSbench.onosUninstall( nodeIp=ONOS5Ip )
> >          main.ONOSbench.onosUninstall( nodeIp=ONOS6Ip )
>
> I did not change any driver files after pulling Kelvins mininet changes
> sometime back. So, this commit should have only
> OnosCHO file changes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opennetworkinglab/ONLabTest/pull/63/files#r24964796>.
>



-- 
Jon Hall
ON.Lab
jhall@onlab.us
